pkgs/by-name/ko/komac/package.nix

```diff
@@ -1,22 +1,25 @@
-{ lib
-, stdenv
-, fetchFromGitHub
-, pkg-config
-, openssl
-, rustPlatform
-, darwin
-, testers
-, komac
-, dbus
-, zstd
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  pkg-config,
+  openssl,
+  rustPlatform,
+  darwin,
+  testers,
+  komac,
+  dbus,
+  zstd,
+  versionCheckHook,
+  nix-update-script,
 }:
 
 let
   version = "2.6.0";
   src = fetchFromGitHub {
     owner = "russellbanks";
     repo = "Komac";
-    rev = "v${version}";
+    rev = "refs/tags/v${version}";
```

Comment:
Agreed to many packages are applying this prefix for the revision number. Is this the recommended format?

Reply:
And this change was looks applied in non related commit https://github.com/NixOS/nixpkgs/pull/348876/commits/a2e768522a3b3f64bee31fac7664361d2eb1da79

Reply:
> Agreed to many packages are applying this prefix for the revision number. Is this the recommended format?

It is just to ensure we are getting from a tag. But it probably is the recommended because some people recommended it to me when I started packaging

Reply:
> And this change was looks applied in non related commit [a2e7685](https://github.com/NixOS/nixpkgs/commit/a2e768522a3b3f64bee31fac7664361d2eb1da79)

I will put in into a separate commit.